### PR TITLE
task(ardrive-web): Add lastBlockHeight to drive PE-87

### PR DESCRIPTION
**Pull request by jdaev**
Adds back lastBlockHeight to fix ArDrive not syncing between instances.

### CODE REVIEW

**Review by fedellen — DISMISSED** [review]
Looks good!

File: lib/blocs/sync/sync_cubit.dart [hunk 1]
```diff
@@ -110,7 +110,12 @@ class SyncCubit extends Cubit<SyncState> {
 
     final entityHistory = await _arweave.getNewEntitiesForDrive(
       drive.id,
-      // We should start syncing from the block after the latest one we already synced from.
+      // Starts syncing from lastBlock -5. 5 is arbitrary, just to make sure it
```
Comment (arielmelendez) [inline lib/blocs/sync/sync_cubit.dart:113]: Nit: the grammar here is a little confusing.

File: lib/blocs/sync/sync_cubit.dart [hunk 2]
```diff
@@ -110,7 +110,12 @@ class SyncCubit extends Cubit<SyncState> {
...
+      // picks up all files 'After' indicates the cursor where it should start
+      // syncing from. For first sync it should be null or an empty string.
+      lastBlockHeight: drive.lastBlockHeight > 5
```
Comment (arielmelendez) [inline lib/blocs/sync/sync_cubit.dart:116]: Could this be simplified and made more consistent by using max(drive.lastBlockHeight - 5, 0)?

File: lib/blocs/sync/sync_cubit.dart [hunk 3]
```diff
@@ -122,6 +127,12 @@ class SyncCubit extends Cubit<SyncState> {
 
     //Handle newEntities being empty, i.e; There's nothing more to sync
     if (newEntities == null || newEntities.isEmpty) {
+      //Reset the sync cursor after every sync to
+      //pick up files from other instances.
```
Comment (arielmelendez) [inline lib/blocs/sync/sync_cubit.dart:131]: Nit: instances of what? sync_cubits? ArDrive apps? etc.?

**Review by arielmelendez — DISMISSED** [review]
Seems ok. Left some minor comments.

File: lib/blocs/sync/sync_cubit.dart (hunk 3 above)
Comment (jdaev) [reply lib/blocs/sync/sync_cubit.dart]: Elaborated [here](https://github.com/ardriveapp/ardrive-web/pull/180/commits/75294c2924ed648266dba463ef1273e06e66f8f1)

File: lib/blocs/sync/sync_cubit.dart (hunk 2 above)
Comment (jdaev) [reply lib/blocs/sync/sync_cubit.dart]: Added [here](https://github.com/ardriveapp/ardrive-web/pull/180/commits/75294c2924ed648266dba463ef1273e06e66f8f1)

File: lib/blocs/sync/sync_cubit.dart (hunk 1 above)
Comment (jdaev) [reply lib/blocs/sync/sync_cubit.dart]: Explained [here](https://github.com/ardriveapp/ardrive-web/pull/180/commits/75294c2924ed648266dba463ef1273e06e66f8f1)

**Review by fedellen — APPROVED** [review]

**Pull request merged**